utils/upload_nuc_data.py

```diff
@@ -0,0 +1,56 @@
+"""# This is a simple tool to update the cyclus_nuc_data.h5 taken from pyne's upload.py
```

Comment:
This file needs to list PyNE's copyright notice or the (notably absent) license file in this repo needs to list this file as being listed originally under the PyNE's copyright.